update test formatting, response to #414

bordeltabernacle opened the pull request.

petertseng commented on exercises/allergies/allergies_test.go:
```diff
@@ -67,9 +67,9 @@ func TestAllergicTo(t *testing.T) {
 		actual := AllergicTo(test.i, test.allergen)
 		if actual != test.expected {
-			t.Fatalf("FAIL: AllergicTo(%d, %s): expected %t, actual %t", test.i, test.allergen, test.expected, actual)
+			t.Fatalf("FAIL: AllergicTo(%d, %q): expected %t, actual %t", test.i, test.allergen, test.expected, actual)
```
this one wouldn't strictly have been necessary since it's just a single string (and we know it will always be non-empty since the test provides it, not the student solution), but won't hurt to have.

petertseng commented on exercises/allergies/allergies_test.go:
```diff
@@ -67,9 +67,9 @@ func TestAllergicTo(t *testing.T) {
+			t.Fatalf("FAIL: AllergicTo(%d, %q): expected %t, actual %t", test.i, test.allergen, test.expected, actual)
 		} else {
-			t.Logf("PASS: AllergicTo(%d, %s) %t", test.i, test.allergen, actual)
+			t.Logf("PASS: AllergicTo(%d, %q) %t", test.i, test.allergen, actual)
```
same: this one wouldn't strictly have been necessary since it's just a single string (and we know it will always be non-empty since the test provides it, not the student solution), but won't hurt to have.

bordeltabernacle commented:
I changed and undid a few in grade-school, seeing what difference it made, I think I `ctrl-z`'d a bit too much there! Anyway, I've fixed it up and changed the `allergies_test` as well. Even though you say it can't hurt, it's not necessary, so probably best to leave as is, plus I learnt how to remove a commit! :smiley:

petertseng commented:
Thanks!
